templates/register_of_disqualifications/list.html.tx

```diff
@@ -11,11 +11,11 @@
                 % for $alphabet -> $letter {
                     % if $active_letter == $letter {
                         <li class="active">
-                            <a class="govuk-link" href="/register-of-disqualifications/<%= $letter %>" class="page"><span class="govuk-visually-hidden">Only show surnames beginning with</span><%= $letter %></a>
+                            <span id="page-1" class="page govuk-link"><%= $letter %></span>
                         </li>
                     % } else {
                         <li class="inactive">
-                            <a class="govuk-link" href="/register-of-disqualifications/<%= $letter %>" class="page"><span class="govuk-visually-hidden">Only show surnames beginning with</span><%= $letter %></a>
+                            <a class="govuk-link  v " href="/register-of-disqualifications/<%= $letter %>" class="page"><span class="govuk-visually-hidden">Only show surnames beginning with</span><%= $letter %></a>
```

Comment:
is this 'v' intentional?

Reply:
No, it's not. I'll have that checked. There should be another class in there. I probably did a cmd v which didn't get pasted. Thanks